### ISSUE TITLE
new penalty suggestion method in jobToRefine?

### ISSUE THREAD

**Issue by tdhock** [PeakLearner/PeakLearner#118]
hi @deltarod I would like to add a new method for determining penalties to compute for contig/tracks where we have labels (and so we would like to learn which penalties are consistent with those labels).
My current plan is to make a new function called submitTargetInterval() in core/Jobs/Jobs.py then call that function from within jobToRefine -- does that sound reasonable?
Two questions.
1. how can I interactively test my new code? Can I do something like `from Jobs import jobToRefine` and then call that function interactively from my python command line? What is the recommended interactive testing approach?
2. The code that I would like to call to determine the new penalties is here, https://github.com/tdhock/model_selection_breakpoints#usage It requires as input a DataFrame which is similar to what we get from the ModelSummary API, but it also has a column for the total un-penalized Poisson loss. How can I construct such a table from within Jobs.py?
Thanks in advance.

**Comment by deltarod** [comment]
> My current plan is to make a new function called submitTargetInterval() in core/Jobs/Jobs.py then call that function from within jobToRefine -- does that sound reasonable?

No, jobToRefine and the whole job spawning system in the current iteration is a bit hacked together and the next iteration should make this significantly easier.

> how can I interactively test my new code?

I don't have an interactive testing approach due to the database and the amount of support which it needs in the current iteration. You should be able to do something like this once I get the refactor done.

> The code that I would like to call to determine the new penalties is here, https://github.com/tdhock/model_selection_breakpoints#usage It requires as input a DataFrame which is similar to what we get from the ModelSummary API, but it also has a column for the total un-penalized Poisson loss. How can I construct such a table from within Jobs.py?

I need to flush out the new jobs system and then I can direct you better.
